pages/library/_viewId.vue

```diff
@@ -48,8 +47,28 @@ export default Vue.extend({
         sortBy: 'SortName',
         sortOrder: 'Ascending'
       });
+
       this.Items = itemsResponse.data.Items;
     }
+  },
+  methods: {
+    imageLink(id: string) {
+      const url = `${this.$axios.defaults.baseURL}/Items/${id}/Images/Primary`;
+      return url;
```

Comment:
You can probably merge these two lines and return the template literal directly.

Reply:
Will do 👍